doc/examples/developers/plot_max_tree.py

```diff
@@ -43,7 +43,7 @@
 import numpy as np
 import matplotlib.pyplot as plt
 from matplotlib.lines import Line2D
-from skimage.morphology import max_tree
+from skimage.morphology.max_tree import max_tree
```

Comment:
I am assuming this is a fix to the "module not callable" error? I really like that you are trying to find solutions yourself! :ok_hand: 

I think we may have hit a similar error before in https://github.com/scikit-image/scikit-image/pull/6983. While this solution may work, it also exposes a submodule that is not part of our API. In scikit-image we usually aim for only exposing two levels of modules: `skimage` and `skimage.morphology`, `skimage.filter`, ... 

So we probably want to revert this change? Than we can have a look how we might fix it without exposing `skimage.morphology.max_tree`.

Reply:
Agreed.

Reply:
@decorouz in case you might be stuck with this? If you want to overcome the "module not callable" error by following the approach taken by @ana42742 in #6983, you should rename `skimage/morphology/max_tree.py` into `skimage/morphology/_max_tree.py` (use `git mv` for this).